ckanext/portalopendatadk/templates/snippets/package_item.html

```
@@ -42,12 +44,16 @@ <h3 class="dataset-heading">
                 <span class="label label-important">{{ _('Deleted') }}</span>
               {% endif %}
               {{ h.popular('recent views', package.tracking_summary.recent, min=10) if package.tracking_summary }}
-	    
+
             {% endblock %}
 		{% if package.organization %}
 			</br>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </br> ]

(tag-pair)